demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java

```diff
@@ -39,7 +39,7 @@ public WebKeyword(WebDriver driver) {
      * @return WebKeyword to set value for element
      */
     public WebKeyword setValueForElement(WebElement webElement, chooseTypeOfSelect type, String value){

```

Comment:
rename to setValueForSelectElement or setValueForDdlElement

Reply:
thanks for feedback, i was rename it.